CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 3.14.0 (2020-02-07)
 
 - Updated `skyux serve` to include the `host` properties from `skyuxconfig.json` in the Host URL.
+- Move help-widget logic into latest `@blackbaud/skyux-lib-help`. [#190](https://github.com/blackbaud/skyux-sdk-builder/pull/190)
```

Comment:
Can you make this past tense?

Reply:
Good call. Also, should "help-widget" be a code snippet? If we aren't referring to code here, we don't need the hyphen. And we're also missing "the" before "latest."